test/controller.js

```diff
@@ -76,4 +76,20 @@ describe('controller', () => {
 
     expect(instance.foo).to.have.been.called(1)
   })
+
+  it('upgrades child decendants when connected', done => {
+    controller(class ChildElementElement extends HTMLElement {})
+    controller(
+      class ParentElementElement extends HTMLElement {
+        connectedCallback() {
+          const child = this.querySelector('child-element')
+          expect(child.matches(':defined')).to.equal(true)
+          done()
+        }
+      }
+    )
+
+    // eslint-disable-next-line github/unescaped-html-literal
+    document.body.innerHTML = '<parent-element><child-element></child-element></parent-element>'
```

Comment:
Can we please use a fixture here, like in https://github.com/github/catalyst/blob/aa425d6936842c1857b5f349fab7b940d43effd4/test/bind.js#L6-L15


```suggestion
    // eslint-disable-next-line github/unescaped-html-literal
    root.innerHTML = '<parent-element><child-element></child-element></parent-element>'
```